.github/CONTRIBUTING.md

```diff
@@ -1,175 +1,152 @@
 
-- Visual Studio C++ 10. Link to download
-- Install CMake >= `<VERSION>` Link to download
-- Python 3.4. Link to download
-- Install the nrtest dependencies. Links to download
+1. how to configure your machine to build the project locally; 
+2. how to obtain the project files using git; 
+3. how to use cmake to generate build files and build the project; and 
+4. how to use ctest and nrtest to perform unit and regression testing on the build artifacts produced. Be advised, you will need local admin privileges on your machine to follow this tutorial. Let’s begin! 
 
-### Building
-To build SWMM then run:
+### Dependencies
 
-```
-$ cmake -G "Visual Studio <VERSION>" -DCMAKE_INSTALL_PREFIX:PATH=output -DCMAKE_BUILD_TYPE:STRING=Release
-```
+Before the project can be built the required tools must be installed. The OWA SWMM project adheres to a platform compiler policy - for each platform there is a designated compiler. The platform compiler for Windows is Visual Studio cl, for Linux gcc, and for Mac OS clang. These instructions describe how to build SWMM on Windows. CMake is a cross platform build, testing, and packaging tool that is used to automate the SWMM build workflow. Boost is a free portable peer-reviewed C++ library. Unit tests are linked with Boost unit test libraries. Lastly, git is a free and open source distributed version control system. Git must be installed to obtain the project source code from the OWA SWMM repository found on GitHub. 
 
-### Running
+### Summary of Dependencies
+- Platform Compiler
+  - Windows: Visual Studio 10.0 32-bit cl (version 16.00.40219.01 for 80x86)
+- CMake (version 3.0.0 or greater)
+- Boost Libraries (version 1.58 or greater)
+- git (version 2.6.0 or greater)
 
-```
-$ run-swmm --help
-```
 
-## B.) Linux
+#### Step 1 - Install Dependencies
+###### Install Visual Studio 2010 Express and SP1
 
-### Install Dependencies
-- Build dependencies... gcc >=  libgcc >=
-- cmake
-- Install the nrtest dependencies..... Links to download
+Our current benchmark platform and compiler is Windows 32-bit Visual Studio 10 2010. Older versions of Visual Studio are available for download here:  
 
-#### On Debian (.deb) based systems you can use
-```
-$ sudo apt install ...
-```
+https://www.visualstudio.com/vs/older-downloads/ 
 
-#### On Centos (.rpm) based systems you can use
+A service pack for Visual Studio 10 2010 is available here: 
 
-```
-$ sudo yum install ...
-```
+https://www.microsoft.com/en-us/download/details.aspx?id=34677
 
-### Building
+###### Install Boost 
 
-```
-$ cmake -DCMAKE_INSTALL_PREFIX=$PREFIX -DCMAKE_BUILD_TYPE:STRING=Release ..
-$ make -j $CPU_COUNT
-$ make install
-```
+Boost binaries for Windows offer a convenient installation solution. Be sure to select for download the boost installer exe that corresponds to the version of Visual Studio you have installed. 
```

Comment:
Should the second sentence say "Be sure to select _download for_ the boost installer...."?